docs/chaos-engineering/chaos-faults/aws/ecs-agent-stop.md

```diff
@@ -0,0 +1,183 @@
+
+- Ensure to create a Kubernetes secret having the AWS access configuration(key) in the `CHAOS_NAMESPACE`. A sample secret file looks like:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-secret
+type: Opaque
+stringData:
+  cloud_config.yml: |-
+    # Add the cloud AWS credentials respectively
+    [default]
+    aws_access_key_id = XXXXXXXXXXXXXXXXXXX
+    aws_secret_access_key = XXXXXXXXXXXXXXX
+```
+
+- If you change the secret key name (from `cloud_config.yml`) please also update the `AWS_SHARED_CREDENTIALS_FILE` ENV value on `experiment.yaml`with the same name.
+
+## Default Validations
+
+:::info
+
+- ECS container instance should be in healthy state.
+
+:::
+
+## Experiment tunables
+
+<details>
+    <summary>Check the Experiment Tunables</summary>
+    <h2>Mandatory Fields</h2>
+    <table>
+        <tr>
+        <th> Variables </th>
+        <th> Description </th>
+        <th> Notes </th>
+        </tr>
+        <tr> 
+        <td> CLUSTER_NAME </td>
+        <td> Name of the target ECS cluster</td>
+        <td>  </td>
+        </tr>
+        <tr>
+        <td> REGION </td>
+        <td> The region name of the target ECS cluster</td>
+        <td> </td>
+        </tr>
+    </table>
+    <h2>Optional Fields</h2>
+    <table>
+      <tr>
+        <th> Variables </th>
+        <th> Description </th>
+        <th> Notes </th>
+      </tr>
+      <tr>
+        <td> TOTAL_CHAOS_DURATION </td>
+        <td> The total time duration for chaos insertion (sec) </td>
+        <td> Defaults to 30s </td>
+      </tr>
+      <tr>
+        <td> CHAOS_INTERVAL </td>
+        <td> The interval (in sec) between successive instance termination.</td>
+        <td> Defaults to 30s </td>
+      </tr>
+      <tr> 
+        <td> AWS_SHARED_CREDENTIALS_FILE </td>
+        <td> Provide the path for aws secret credentials</td>
+        <td> Defaults to <code>/tmp/cloud_config.yml</code> </td>
+      </tr>
+      <tr> 
+        <td> DOCUMENT_NAME </td>
+        <td> Provide the name of addded ssm docs (if not using the in-build docs)</td>
+        <td> Default to Litmus-AWS-SSM-Docs-For-CPU-Hog</td>
+      </tr>
+      <tr> 
+        <td> DOCUMENT_FORMAT </td>
+        <td> Provide the format of the ssm docs. It can be YAML or JSON</td>
+        <td> Defaults to <code>YAML</code> </td>
+      </tr>
+      <tr> 
+        <td> DOCUMENT_TYPE </td>
+        <td> Provide the document type of added ssm docs (if not using the default docs)</td>
+        <td> Defaults to <code>Command</code> </td>
+      </tr>
+      <tr> 
+        <td> DOCUMENT_PATH </td>
+        <td> Provide the document path if added using configmaps</td>
+        <td> Defaults to Litmus-AWS-SSM-Docs-For-CPU-Hog.yml </td>
+      </tr>
+      <tr> 
+        <td> INSTALL_DEPENDENCIES </td>
+        <td> Select to install dependencies used to run the cpu chaos. It can be either True or False</td>
+        <td> Defaults to True </td>
+      </tr>
```

Comment:
as suggested by @avaakash "We can remove these as the default document will be used for these experiments, if the user wants to use a custom document they can use the SSM Command experiment ( as discussed recently)"